### PR TITLE
TreeComboBox

### PR DESCRIPTION
**Pull request by rabbitism**

### CODE REVIEW

File: src/Ursa.Themes.Semi/Controls/TreeComboBox.axaml [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+                        IsLightDismissEnabled="True"
+                        IsOpen="{Binding IsDropDownOpen, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
+                        PlacementTarget="Background"
+                        WindowManagerAddShadowHint="False">
+                        <Border
+                            Name="PopupBorder"
+                            Margin="0,4"
+                            Background="{DynamicResource ComboBoxPopupBackground}"
+                            BorderBrush="{DynamicResource ComboBoxPopupBorderBrush}"
+                            BoxShadow="{DynamicResource ComboBoxPopupBoxShadow}"
```
Comment (zdpcdt) [inline src/Ursa.Themes.Semi/Controls/TreeComboBox.axaml:118]: Missing BorderThickness

**Comment by zdpcdt** [comment]
It seems that the part of InputBox miss the :pressed Classes.

File: src/Ursa.Themes.Semi/Controls/TreeComboBox.axaml (hunk 1 above)
Comment (zdpcdt) [reply src/Ursa.Themes.Semi/Controls/TreeComboBox.axaml:118]: The BoxShadow is not obvious.

Comment (rabbitism) [reply src/Ursa.Themes.Semi/Controls/TreeComboBox.axaml:118]: Added BorderThickness. shadow is also not obvious in combobox, will fix with that together if any idea.

**Comment by rabbitism** [comment]
Updated.

File: src/Ursa.Themes.Semi/Controls/TreeComboBox.axaml (hunk 1 above)
Comment (zdpcdt) [reply src/Ursa.Themes.Semi/Controls/TreeComboBox.axaml:118]: I think it's necessary to fix it.

**Review by zdpcdt — APPROVED** [review]

**Pull request merged**